### PR TITLE
Release 0.6.0-rc1

### PR DESCRIPTION
**Pull request by ueco-jb**
closes https://github.com/confio/poe-contracts/issues/88

### CODE REVIEW

**Review by ethanfrey — APPROVED** [review]
Looks good, but can you get in #85 before you merge.

(And did you check MIGRATING.md is updated, maybe already in the previous PRs, but I think there were some recent changes)

**Comment by maurolacy** [comment]
And perhaps #96 as well?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,29 @@
...
+
+**Merged pull requests:**
+
+- Valset: Implement ListJailedValidators query [\#92](https://github.com/confio/poe-contracts/pull/92) ([ueco-jb](https://github.com/ueco-jb))
+- Valset: metadata validation [\#84](https://github.com/confio/poe-contracts/pull/84) ([ueco-jb](https://github.com/ueco-jb))
+- Increase MAX\_LIMIT constant to 100 and DEFAULT\_LIMIT to 30 [\#80](https://github.com/confio/poe-contracts/pull/80) ([ueco-jb](https://github.com/ueco-jb))
+- Voting Contract: Add list\_votes\_by\_voter query [\#79](https://github.com/confio/poe-contracts/pull/79) ([ueco-jb](https://github.com/ueco-jb))
+- Feature-gated `SimulateValidatorSet` [\#77](https://github.com/confio/poe-contracts/pull/77) ([maurolacy](https://github.com/maurolacy))
+- Add `diff_schema.sh` / `diff_state.sh` tools [\#74](https://github.com/confio/poe-contracts/pull/74) ([maurolacy](https://github.com/maurolacy))
+- Rename API to points and rewards [\#50](https://github.com/confio/poe-contracts/pull/50) ([ethanfrey](https://github.com/ethanfrey))
```
Comment (maurolacy) [inline CHANGELOG.md:28]: This should be tagged as / moved to Breaking. Perhaps other as well?

Comment (ueco-jb) [reply CHANGELOG.md]: Good point, both included

**Pull request merged**